source/docs/software/state-space/index.rst

```diff
@@ -0,0 +1,11 @@
+State-Space and Model Based Control with WPILib
```

Comment:
This should be under advanced-controls.

Reply:
Should state space be a subfolder under advanced controls?

Reply:
Yes